auth/users.yaml

```diff
@@ -4,6 +4,11 @@ users:
     groups:
       - maintainer
 
+  - name: tle
```

Comment:
Do you mind moving your entry to the end of the file? The uid's are currently generated in sequential order starting from 2001. Inserting a user ahead of others will cause to the uid's underneath your entry to change, which makes a headache for fixing file permissions.